webapp/app/plugins/controller.js

```diff
@@ -124,7 +124,9 @@ treeherder.controller('PluginCtrl', [
                         }, []);
                     }
                     // the fourth result comes from the jobLogUrl artifact
-                    $scope.job_log_urls = results[3];
+                    // filter to only the builds-4h logs
+                    $scope.job_log_urls = _.where(results[3], {name: 'builds-4h'});
```

Comment:
We'll need to change the name here if we make the s/build-4h/.../ change - but for now looks great :-)